core/src/test/java/io/grpc/internal/ServerImplTest.java

```diff
@@ -407,8 +406,11 @@ public void basicExchangeSuccessful() throws Exception {
     assertNotNull(statsTraceCtx);
     when(stream.statsTraceContext()).thenReturn(statsTraceCtx);
 
-    ServerStreamListener streamListener
-        = transportListener.streamCreated(stream, "Waiter/serve", requestHeaders);
+    transportListener.streamCreated(stream, "Waiter/serve", requestHeaders);
+    ArgumentCaptor<ServerStreamListener> streamListenerCaptor =
```

Comment:
Since a captor for this type is used frequently, feel free to make it a field in the class like `statusCaptor` (possibly with `@Captor`).